sql.go

```diff
@@ -0,0 +1,158 @@
+package scim
+
+import (
+	"fmt"
+	"strings"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/di-wu/scim-filter-parser"
+	filter "github.com/di-wu/scim-filter-parser"
```

Comment:
Is line 8 needed if line 9 imports it as `filter`? This might be a Go thing I don't grok.

Reply:
Nope that was my IDE trying to make sense of what I was writing lol. One spot I used it as `scim`. Wanted to use `filter` to avoid confusion. Good catch! 4b86d05554c489342163e040d42f9ee41579ad67